src/main/scala/language.scala

```diff
@@ -0,0 +1,51 @@
+package piconot
+import scalafx.application.JFXApp
+import picolib.semantics._
+/**
+ * @author justisallen
+ */
+trait language extends JFXApp{
+ 
+  val blank = Surroundings(Anything,Anything,Anything,Anything)
+  val No = Surroundings(Open,Anything,Anything,Anything)
+  val Nx = Surroundings(Blocked,Anything,Anything,Anything)
+  val So = Surroundings(Anything,Anything,Anything,Open)
+  val Sx = Surroundings(Anything,Anything,Anything,Blocked)
+  val Eo = Surroundings(Anything,Open,Anything,Anything)
+  val Ex = Surroundings(Anything,Blocked,Anything,Anything)
+  val Wo = Surroundings(Anything,Anything,Open,Anything)
+  val Wx = Surroundings(Anything,Anything,Blocked,Anything)
```

Comment:
We _do_ use the values in `and`, but only to match on the input. The reason we don't use the values again for the output is because we want to maintain the state of the `toReturn` 'accumulator,' which would otherwise be overwritten in a sense if we just returned one of the defined values.